### PR TITLE
height/width loader spinner centered

### PR DESCRIPTION
**Pull request by SamirTouira**
related to: #4769 

made with @RemcoSimonides

### CODE REVIEW

File: apps/festival/festival/src/app/marketplace/event/session/session.component.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h2>by {{ event.movie.directors | displayName }}</h2>
 
 
 <ng-template #loading>
-  <section fxLayout="column" fxLayoutAlign="center center">
+  <section class="loading-spinner" fxLayout="column" fxLayoutAlign="center center">
```
Comment (fritzschoff) [inline apps/festival/festival/src/app/marketplace/event/session/session.component.html:65]: guess you don't need the flex layout anymore if you are using position absolute?

**Review by fritzschoff — APPROVED** [review]

File: apps/festival/festival/src/app/marketplace/event/session/session.component.scss [hunk 2]
```diff
@@ -92,6 +92,15 @@ p {
   }
 }
 
+.loading-spinner {
+  position:absolute;
+	left:0;
+	right:0;
+	top:0;
+	bottom:0;
+	margin:auto;
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/marketplace/event/session/session.component.scss:101]: You can remove `margin: auto`

**Review by RemcoSimonides — CHANGES_REQUESTED** [review]
Wait

**Review by RemcoSimonides — APPROVED** [review]

File: apps/festival/festival/src/app/marketplace/event/session/session.component.html (hunk 1 above)
Comment (SamirTouira) [reply apps/festival/festival/src/app/marketplace/event/session/session.component.html:65]: I can remove the `fxLayout="column"` indeed, but I have to keep `fxLayoutAlign="center center"` to keep the spinner centered. I just let this if you are okay

**Pull request merged**